src/aleph/sdk/client/http.py

```diff
@@ -48,6 +50,8 @@ def __init__(
         if unix_socket_path and allow_unix_sockets:
             check_unix_socket_valid(unix_socket_path)
             connector = aiohttp.UnixConnector(path=unix_socket_path)
+        elif ssl_context:
+            connector = aiohttp.TCPConnector(ssl=ssl_context)
```

Comment:
A potential issue I see here is that we create two connectors (first `UnixConnector` and then `TCPConnector`), if we pass `ssl_context` into the __init__ function, due to the way the if/elif is setup. You can avoid that with:
```suggestion
        if ssl_context:
            connector = aiohttp.TCPConnector(ssl=ssl_context)
        elif unix_socket_path and allow_unix_sockets:
            check_unix_socket_valid(unix_socket_path)
            connector = aiohttp.UnixConnector(path=unix_socket_path)
```